src/components/pages/observation/list/views/stats/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { SimpleGrid } from "@chakra-ui/react";
+import useObservationFilter from "@components/pages/observation/common/use-observation-filter";
+import React from "react";
+
+import SpeciesGroups from "./species-groups";
+import StatesDistribution from "./states-distribution";
+
+export default function StatsView() {
+  const { observationData, speciesGroup, filter } = useObservationFilter();
+
+  return (
+    <SimpleGrid minChildWidth={10} spacing={4}>
```

Comment:
Why there's need of `minChildWidth` prop if you are trying to achieve two width layout you can provide columns in  `<SimpleGrid/>` component, this might help

https://chakra-ui.com/docs/layout/simple-grid